### PR TITLE
4 map participants to their answers

### PR DESCRIPTION
**Pull request by leonardk1**
#4
As An Admin
I Want To Map the Participants to the the question and answers they gave in a given campaign
So That I can be able to make the necessary data analyses depending on their data

#### Tasks
 - [x] create the `participant_answer` model

#### Notes
 this model is simply keeping records that show which options were chosen by the different participants per questions

### CODE REVIEW

**Comment by leonardk1** [comment]
@godfreym please review

**Comment by leonardk1** [comment]
@godfreym please review

File: app/models/question.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class Question < ActiveRecord::Base
   has_many :options, dependent: :delete_all
+  has_many :participant_answers, dependent: :delete_all
```
Comment (godfreym) [inline app/models/question.rb]: @leonardk1 does question has many answers or just a single answer for each question ?

Comment (leonardk1) [inline app/models/question.rb]: Which year were you born?
1990
1992
@godfreym 
so a question has many options(answers)
what are `participant_answers`?
these are the records to show which option the participant chose per question

Comment (godfreym) [inline app/models/question.rb]: @leonardk1 OK cool

**Comment by leonardk1** [comment]
@arafatm please review

**Comment by arafatm** [comment]
:+1:

**Pull request merged**